lib/global-styles.php

```diff
@@ -135,16 +134,48 @@ function_exists( 'gutenberg_is_edit_site_page' ) &&
 		$settings['styles'][] = $block_styles;
 	}
 
+	// Copied from get_block_editor_settings() at wordpress-develop/block-editor.php.
```

Comment:
What's this comment about? It's not clear to me :)

Reply:
This is just a note so we remember that this part has landed in WordPress core. We're still re-computing all the settings from the plugin, so we need this code as well. I thought it can be helpful to our future selves.